js/main.js

```diff
@@ -1,2 +1,29 @@
 /* global data */
 /* exported data */
+var $form = document.querySelector('form');
+$form.addEventListener('submit', submitButton);
+
+var $photo = document.querySelector('img');
+var $photoURL = document.querySelector('#photoURL');
+$photoURL.addEventListener('input', updateURL);
+
+function updateURL(event) {
+  var src = $photoURL.value;
+  $photo.setAttribute('src', src);
+}
+
+function submitButton(event) {
+  event.preventDefault();
+  data.nextEntryId += 1;
+  var inputObj = {
+    title: $form.elements.title.value,
+    imageURL: $form.elements.photoURL.value,
+    notes: $form.elements.notes.value,
+    nextEntryId: data.nextEntryId
```

Comment:
`nextEntryId` is the name of the property in the data model, because it holds the next id to use for an entry. In this object, you are just giving the entry an id, so `nextEntryId` isn't an accurate name. You should name the property in this object something like `entryId`